server/db/models.py

```diff
@@ -256,8 +256,11 @@
 
 map_pool_map_version = Table(
     "map_pool_map_version", metadata,
-    Column("map_pool_id",       Integer, ForeignKey("map_pool.id"),     nullable=False),
-    Column("map_version_id",    Integer, ForeignKey("map_version.id"),  nullable=False),
+    Column("id",            Integer,        primary_key=True),
+    Column("map_pool_id",   Integer,        ForeignKey("map_pool.id"),      nullable=False),
+    Column("map_version_id", Integer,       ForeignKey("map_version.id")),
```

Comment:
Can we line these up
```suggestion
    Column("map_pool_id",    Integer,       ForeignKey("map_pool.id"),      nullable=False),
    Column("map_version_id", Integer,       ForeignKey("map_version.id")),
```